app/assets/stylesheets/modules/shared/email_status.scss

```diff
@@ -36,7 +42,7 @@
 
     &.last {
       img.approved.linear, img.pending.linear, img.rejected.linear {
-        background-image: none;
+        background-image: none !important;
```

Comment:
Is there a way to get what you need without the `!important`? Generally should be avoided